WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -384,9 +382,8 @@ public boolean onPreferenceClick(Preference preference) {
         } else if (preference == mCategoryPref || preference == mFormatPref) {
             return !shouldShowListPreference((DetailListPreference) preference);
         } else if (preference == mExportSitePref) {
-            showExportContentDialog();
         } else if (preference == mDeleteSitePref) {
-            showDeleteSiteDialog();
+            requestPurchasesForDeletionCheck();
         } else {
```

Comment:
I know this is not in the changes, but this `else` here is not necessary, I think we should just `return false` in the end. Since it's a very small change, can we add it to this PR?